packages/react-components/src/components/DevicePermissions/DomainPermissionsScaffolding.tsx

```diff
@@ -122,7 +122,10 @@ export const DomainPermissionsContainer = (props: DomainPermissionsContainerProp
   const theme = useTheme();
 
   return (
-    <Stack style={{ padding: '2rem', maxWidth: '25.375rem' }} aria-label={strings?.ariaLabel}>
+    <Stack
+      style={{ padding: '2rem', maxWidth: '25.375rem', maxHeight: '100vh', alignSelf: 'center', overflow: 'hidden' }}
```

Comment:
In fact this _should_ have a scrollbar because the Need help link cannot be accessed:
![image](https://user-images.githubusercontent.com/2684369/206313098-b090d04d-6373-4a95-a8b4-cee9c7695b96.png)

Reply:
Yeah you are right, we should not have overflow: hidden on, will remove that